app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -244,7 +243,6 @@ def handle_unsupported_id_type(enrollment, response)
     analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
       **enrollment_analytics_attributes(enrollment, complete: true),
       **response_analytics_attributes(response),
-      fraud_suspected: response['fraudSuspected'],
```

Comment:
is this duplicated somewhere else?

Reply:
yeah, it's in `response_analytics_attributes`